client/src/client_sync/v21/wallet.rs

```diff
@@ -10,6 +10,9 @@
 //! See or use the `define_jsonrpc_minreq_client!` macro to define a `Client`.
 
 /// Implements Bitcoin Core JSON-RPC API method `createwallet` with descriptors=true (descriptor wallet)
+///
+/// This is a helper function only needed in v21 and v22 to create a wallet with descriptors.
+/// From v23 onwards descriptors=true is the default.
 #[macro_export]
 macro_rules! impl_client_v21__create_wallet_with_descriptors {
```

Comment:
Unless my git-foo is letting me down this isn't called anymore? What happened the test code that used it?